lib/mutant/ast/named_children.rb

```diff
@@ -13,8 +13,6 @@ module NamedChildren
       # @api private
       #
       def self.included(host)
-        super
```

Comment:
I could not come up with a test that kills this mutation and gave up. As I control the environment here and `NamedChildren` will not be called outside of mutant I'm fine to fix this when it becomes a problem. If I'd provide an public interface where I ask my users to include a module of mine that defines `.included` I'd keep it.